### ISSUE TITLE
Make a proper documentation PLEASE!

### ISSUE THREAD

**Issue by Lotar122** [ncsoft/Unreal.js#330]
plugin seams to work okay but in 5.0.3 i can't acces any object and i think that a solid documentation would solve that issue youtube tutorials are a piece of junk i can't access my blueprint classes nor the GWorld

DO THE DOCUMENTATION MAN!!!

**Comment by zacharygriffee** [comment]
I agree. I see the project is still being maintained, but much of the documentation is almost ten years old. I've come back to this repo multiple times trying to decide if I should implement it into my projects or not. I feel like UnrealJS would help a lot to bridge the gap between many of the immerging web technologies and the unreal engine, but the documentation is pretty rough to work with. I've spent months couple years back digging through the unrealjs code and what it can do under the hood, and the documentation doesn't even address 1% of what it itself can do. UnrealJS is super powerful stuff, and documentation would make incorporating it into my projects much easier.
